src/migrations/migration_steps.ts

```diff
@@ -538,11 +538,24 @@ migrationStepRegistry
     },
   })
   .add("18.5.1", {
-    migrate(data: WorkbookData): any {
+    migrate(data: any): WorkbookData {
```

Comment:
Could you add another step instead of changing this one ?
because dashboards in odoo need to go through this upgrade moulinette